### PR TITLE
config template #7 first try

### PR DESCRIPTION
**Pull request by andreygalizin**
I doubt that I clearly understand all tasks, let it will be the first try.

### CODE REVIEW

File: roles/mytd/tasks/change_config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (jollymover) [inline roles/mytd/tasks/change_config.yml]: It makes sense to collect all the tasks in a single nginx.yml

File: roles/mytd/tasks/copy_config.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+- name: copy template to conf
```
Comment (jollymover) [inline roles/mytd/tasks/copy_config.yml]: Start the name of the task with the name of the tasks's file. For example:

```yml
name: copy_config | Copy template to conf
```

File: roles/mytd/templates/nginx_default_config.j2 [hunk 3]
```diff
@@ -0,0 +1,85 @@
+user www-data;
```
Comment (jollymover) [inline roles/mytd/templates/nginx_default_config.j2]: * Use the `/etc/nginx/sites-available/default` for templating
* Define playbook parameter `port`
* Use the `port` parameter in the template

**Review by jollymover — CHANGES_REQUESTED** [review]

File: roles/nginx/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (jollymover) [inline roles/nginx/tasks/main.yml:1]: Tasks should be in nginx.yml, main.yml should import nginx.yml

**Review by jollymover — CHANGES_REQUESTED** [review]

File: roles/nginx/tasks/main.yml (hunk 4 above)
Comment (andreygalizin) [reply roles/nginx/tasks/main.yml:1]: Import is done

File: roles/nginx/handlers/main.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+- name: nginx_config_validate
```
Comment (jollymover) [inline roles/nginx/handlers/main.yml]: Do you really need this handler?

File: roles/nginx/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+- name: install_nginx | Install and Configure Nginx on Ubuntu
```
Comment (jollymover) [inline roles/nginx/tasks/main.yml]: Don't need name here, remove the line

File: roles/nginx/tasks/nginx.yml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+- name: install_nginx | Install nginx
```
Comment (jollymover) [inline roles/nginx/tasks/nginx.yml]: prefix should be equal the file name: `- name: nginx | Install nginx`

File: roles/nginx/templates/nginx_default_conf.j2 [hunk 8]
```diff
@@ -0,0 +1,85 @@
+user www-data;
```
Comment (jollymover) [inline roles/nginx/templates/nginx_default_conf.j2]: Do you really need the file?

File: roles/nginx/tasks/page.yml [hunk 9]
```diff
@@ -0,0 +1,11 @@
+---
+- name: Remove file (delete file)
```
Comment (jollymover) [inline roles/nginx/tasks/page.yml]: Add the prefix `page |`

File: roles/nginx/tasks/page.yml [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+  ansible.builtin.file:
+    path: /var/www/html/index.nginx-debian.html
+    state: absent
+- name: Copy file
```
Comment (jollymover) [inline roles/nginx/tasks/page.yml]: Each task should be separated by empty line

**Review by jollymover — CHANGES_REQUESTED** [review]

File: roles/nginx/handlers/main.yml (hunk 5 above)
Comment (andreygalizin) [reply roles/nginx/handlers/main.yml]: Excessive handler erased.

**Pull request merged**